extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/PostgreSQLGuacamoleProperties.java

```diff
@@ -92,6 +92,30 @@ private PostgreSQLGuacamoleProperties() {}
 
     };
 
+    /**
+     * Sets the number of seconds the driver will wait for
+     * a response from the database.
+     */
+    public static final IntegerGuacamoleProperty
+            POSTGRESQL_DEFAULT_STATEMENT_TIMEOUT = new IntegerGuacamoleProperty(){
+
+        @Override
+        public String getName() { return "postgresql-default-statement-timeout"; }
+
+    };
+
+    /**
+     * Sets the number of seconds the driver will wait in a read() call
```

Comment:
As above, this isn't setting anything, so more direct language would probably be better, here.